src/components/form/HdInputPhone.vue

```diff
@@ -0,0 +1,383 @@
+<template>
+  <div class="phone-input">
+    <button
+      class="phone-input__selector"
+      aria-haspopup="listbox"
+      @click="toggleDropdown"
+      :aria-label="t.FORM.PHONE.LABEL"
+      :aria-activedescendant="activeCountryCodeAndName"
+    >
+      <span :class="selectedFlagClassNames"></span>
+      <HdIcon
+        width="24px"
+        height="24px"
+        :src="smallArrowIcon"
+        :class="arrowClassNames"
+      />
+    </button>
+    <ul
+      tabindex="-1"
+      role="listbox"
+      v-show="showDropdown"
+      class="phone-input__dropdown"
+      :aria-label="t.FORM.PHONE.LABEL"
+    >
+      <template>
+        <li
+          v-for="(country, index) in sortedCountriesList"
+          :key="country.code"
+          :id="country.code"
+          class="phone-input__dropdown__option"
+          role="option"
+          :aria-selected="selectedCountry.code === country.code"
+          :ref="country.code"
+        >
+          <button @click="selectCountry(country)" class="option">
+            <span :class="getOptionFlagClassNames(country.code)"></span>
+            <p>
+              <span class="option__name">{{t.COUNTRIES[country.code]}}</span>
+              <span class="option__dial-code">{{country.dial_code}}</span>
+            </p>
+          </button>
+          <hr v-if="index === preferredCountries.length - 1" class="phone-input__dropdown__divider" />
+        </li>
+      </template>
+    </ul>
+    <HdInputFormatter
+      ref="input"
+      type="tel"
+      name="phoneNumber"
+      label="Telefonnumer"
+      v-model="phoneNumber"
+      class="phone-input__field"
+      placeholder="+55 555 555 5555"
```

Comment:
It might be confusing if this input would be used only for German numbers, but the intention is to use international numbers too, that's why we used a generic placeholder... but now that you mention it, I can use the selected dial code to modify the placeholder too, and it will be updated everytime we select another one 👍🏼